### PR TITLE
build: rename android package

### PR DESCRIPTION
**Pull request by niall-shaw**
#57 
Renames android package from default `com.reactlibrary` to `org.hyperledger.indy.sdk.reactnative`

Signed-off-by: Niall Shaw <niall.shaw@absa.africa>

### CODE REVIEW

**Review by jakubkoci — APPROVED** [review]
Thanks Niall 👍

**Comment by niall-shaw** [comment]
@TimoGlastra can we create a release that contains this?

**Comment by TimoGlastra** [comment]
There's two things that I would like to check before merging:
1. What do we want to do with #56 -- I added a comment that I think could work well
2. I need to check whether this approach works with https://github.com/animo/indy-sdk-expo-plugin. This is more of a problem for us, so happy to release it already an figure out afterwards, but I don't want to make the expo setup harder

So basically my answer would be to wait for #56 to be merged and resolved

**Comment by JamesKEbert** [comment]
I would tend to agree with Timo that resolving #56 would be ideal first, but I think hopefully that should be doable without too much delay.

**Pull request merged**
